### ISSUE TITLE
249: sim/weather/wind_direction_degt vs sim/weather/wind_direction_degt[n] is a problem

### ISSUE THREAD

**Issue by tngreene** [X-Plane/XPlane2Blender#436]
Related to the cloud case issue from #352, two stupid horrible aweful but real and in the DataRefs.txt file (still!) are

`sim/weather/wind_direction_degt	float	n	[0-359)	The effective direction of the wind at the plane's location.`
and
`sim/weather/wind_direction_degt[0]	float	y	[0	- 360)			The direction the wind is blowing from in degrees from true north c lockwise.` (also [1] and [2])

In the dictionary they are represented as
```
'sw_wnddrctndgt',    ('sim/weather/wind_direction_degt', 1)
'sw_wnddrctndgt[0]', ('sim/weather/wind_direction_degt[0]', 1)
'sw_wnddrctndgt[1]', ('sim/weather/wind_direction_degt[1]', 1)
'sw_wnddrctndgt[2]', ('sim/weather/wind_direction_degt[2]', 1)
'wind_direction_degt',    ('sim/weather/wind_direction_degt', 1)
'wind_direction_degt[0]', ('sim/weather/wind_direction_degt[0]', 1)
'wind_direction_degt[1]', ('sim/weather/wind_direction_degt[1]', 1)
'wind_direction_degt[2]', ('sim/weather/wind_direction_degt[2]', 1)
```

When the converter goes to look up the dataref, it uses
```
            bone_name = bone.name.split('.')[0].strip() # type: BoneName
            bone_name_no_idx = no_idx(bone_name) # type: Union[SName,TailName]

            print("\nLooking up dataref from '{}' and '{}'".format(bone_name, bone_name_no_idx))
            lookup_result = lookup_dataref(bone_name, bone_name_no_idx)
```

If `wind_drection_degt[0]` is used as a bone name, this code finds two different datarefs!
`The effective direction of the wind at the plane's location` vs `The direction the wind is blowing from in degrees from true north c lockwise`

This breaks a major assumption about dataref lookup: A bone name represents 0 or 1 datarefs. By looking up `bone_name_no_idx` so niavely, I have changed the semantics of the dataref.

And so we get `AssertionError: good tailname lookup (('sim/weather/wind_direction_degt', 1)) != good sname lookup (('sim/weather/wind_direction_degt[0]', 1)`

Resolution:

- [x] Make note in Wiki about the above major assumption?
- [ ] 2.49 obviously doesn't disambiguate the way we do it. We could change the lookup system for cases like this where multiple tailnames without idx match or just make a special case for this and any other cases
- [x] Investigate if this appears in any other places (I don't think it does)

**Comment by tngreene** [comment]
Right now we are simply hardcoding this special case in and hoping that no one made a custom DataRef in the DataRefs,txt

**Comment by tngreene** [comment]
This seems like such an edge case I'm saying that I've done what I can do for this. Users can also fix this in 2.79 afterwards
